_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,15 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added readonly attribute to aria-disabled.
+    summaryAdditional: Now, the input element and calendar icon remain discoverable but disabled behaviors are replicated.
+    affectsAccessibility: true
+    affectsJavascript: true
+    affectsMarkup: true
+    githubPr: #6013
```

Comment:
We should remove the `#` here.

It's causing changelog to not include the link to the PR

![image](https://github.com/user-attachments/assets/15fcd61e-ac39-49c6-8bc2-5cb7cb04ecd3)

```suggestion
    githubPr: 6013
```

Reply:
Resolved in [fefb694](https://github.com/uswds/uswds-site/pull/2794/commits/fefb6949907954544486e519967cf3c9c5b7f530)